### ISSUE TITLE
Grid Margins and Outer Margins

### ISSUE THREAD

**Issue by spentacular** [Hammerspoon/hammerspoon#1174]
I may have overlooked it in the docs, but I'm having trouble setting the outer margins vs the grid margins. The basic margins settings give me windows like this:

![screen shot 2016-12-30 at 7 10 10 am](https://cloud.githubusercontent.com/assets/1043478/21565564/40060d6e-ce5f-11e6-92e6-813c71471bca.png)

Is there a way to make it so all the margins between windows and edges is the same width?

My config setup is here as well for reference:

https://github.com/spentacular/dotfiles/blob/master/hammerspoon/init.lua

Thanks!

**Comment by szymonkaliski** [comment]
It's bothering me too, I'm overriding `hs.grid.set` in my config: https://github.com/szymonkaliski/Dotfiles/blob/master/Dotfiles/hammerspoon/overrides.lua (I think I found the original calculations somewhere else, but don't remember the source)

**Comment by jimeh** [comment]
It really bothered me too. @szymonkaliski I hope you don't mind that I borrowed part of the solution you posted in my own config: https://github.com/jimeh/dotfiles/tree/master/hammerspoon/ext

I'm basically downloading and patching a copy of the grid extension and including that in my own setup. I've automated the process via a [Makefile](https://github.com/jimeh/dotfiles/blob/master/hammerspoon/Makefile) for now. But I intend to submit a pull request against Hammerspoon itself when I have some more free time.
